docs/build/Build-for-RPi3-Tizen.md

```diff
@@ -105,8 +120,8 @@ sdb push ./test/run_pass/test_console.js /home/owner/iotjs/
 sdb root on
```

Comment:
I like your convention of using (ubuntu) and (target) to specify on which machine I must type the command. Could you please add `(ubuntu) ` in front of sdb command? Also, for consistency, it would be better to change `(target)$` to `(target) $`. Please notice 1 space.